### ISSUE TITLE
Multiple segments generated at night

### ISSUE THREAD

**Issue by thohug** [datamapio/posmo_one_testing#42]
Zurich, Android

wifi_fence = 1
wifi_over_gps = 1
data_over_wifi_only = 1
do_not_track = 0

---
I have been getting more and more nights that look like this; problem starts as soon as flight mode is turned on:

![grafik](https://user-images.githubusercontent.com/67195528/112266719-b5da2700-8c74-11eb-8b49-7c2226a08032.png)

The amount of cleaning necessary is quite large and probably can't be asked of a standard user...

**Comment by rogerfischer** [comment]
Looks strange with Wi-Fi Fence on. Do you have more than one Wi-Fi?
@Fangio92 any idea what could be the reason for this?

**Comment by Fangio92** [comment]
Is this something that started recently? Did you use flight mode earlier and did you have problem then?

**Comment by thohug** [comment]
No, that has definitely just come up recently; it has been happening occasionly since beginning of mars.

**Comment by Fangio92** [comment]
I would say that this is related to https://datamap.atlassian.net/browse/POPV-32?atlOrigin=eyJpIjoiOTNhM2E1Yjc0YTJlNGMwYmFjZTYwNGE3Y2Y4NmI2OWIiLCJwIjoiaiJ9
And because I didn't change anything for a long time regarding Wifi fence and airplane mode so I would assume that this is backend problem, but if you can and if you have logs sent it to me to verify this assumption.

![3b0035ac-1cc9-493a-8184-8a33946e67b6](https://user-images.githubusercontent.com/11587927/112345015-f3ff3700-8cc4-11eb-93c8-c599797bfb92.png)

**Comment by thohug** [comment]
I observed this recently (sorry video is too big to upload): https://urbanista-my.sharepoint.com/:v:/g/personal/hug_urbanista_ch/EZStvgwAXnxMi5a3RsL1nd4B0Y8QIgztikv5DCRqjDNVUA?e=0jnaug

might be a reason for this behaviour at night...

**Comment by Fangio92** [comment]
Could be. I will investigate.

**Comment by Fangio92** [comment]
Loop wifi fence solved in 333. Please check if this solve your problem with data
